pwndbg/commands/ida.py

```diff
@@ -110,20 +112,23 @@ def save_ida():
     # Remove old version
     os.unlink(full_path)
 
+
 class ida(gdb.Function):
     """Evaluate ida.LocByName() on the supplied value.
     """
+
     def __init__(self):
         super(ida, self).__init__('ida')
+
     def invoke(self, name):
-        name   = name.string()
+        name = name.string()
         result = pwndbg.ida.LocByName(name)
 
-        if 0xffffe000 <= result <= 0xffffffff \
-        or 0xffffffffffffe000 <= result <= 0xffffffffffffffff:
+        if 0xffffe000 <= result <= 0xffffffff or 0xffffffffffffe000 <= result <= 0xffffffffffffffff:
```

Comment:
Meh, probably just easiest to break this into two statements for readability.  Don't care enough to figure out how to make PEP8 happy and the code also be readable.